lib/puppet/provider/package/cisco.rb

```diff
@@ -160,7 +166,7 @@ def in_guestshell?
 
   def install
```

Comment:
I don't see where this method is calling XR-specific logic - it seems like it would fall through to the 'else' case below and try and use 'yum' to install the package which is wrong, right?